centreon/www/front_src/src/ResourceAccessManagement/AddEditResourceAccessRule/FormInputs/components/ActivateSwitch.tsx

```diff
@@ -0,0 +1,57 @@
+import { FormikValues, useFormikContext } from 'formik';
+import { useTranslation } from 'react-i18next';
+
+import { styled } from '@mui/material/styles';
+import { FormControlLabel, Switch as MUISwitch, Tooltip } from '@mui/material';
+
+import {
+  labelActiveOrInactive,
+  labelDisabled,
+  labelEnabled
+} from '../../../translatedLabels';
+
+const Switch = styled(MUISwitch)(({ theme }) => ({
+  '& .MuiSwitch-switchBase': {
+    '&.Mui-checked': {
+      '& + .MuiSwitch-track': {
+        backgroundColor: theme.palette.success.main,
+        opacity: 1
+      },
+      color: theme.palette.common.white
+    }
+  },
+  '& .MuiSwitch-thumb': {
+    backgroundColor: theme.palette.common.white
+  }
+}));
```

Comment:
move this to styles file

Reply:
Done

Reply:
why the is not visible in the demo ? is not implemented yet ?

Reply:
It is visible, it's in Rule Properties section, under Description text area

Reply:
I meant in the Listing

Reply:
It'll be implemented with PATCH endpoint